uiowa/uiowa
# Grad Admissions AOS Updates - round deux

Closes #4706 
Closes #4718 


# How to test


- blt ds --site=grad.admissions.uiowa.edu
- drush @admissionsgrad.local cim
- yarn workspace grad_admissions_core build
- drush @admissionsgrad.local cr
- [ ] Make sure button styles for cost estimate reference on Area of Study display as links (not a button)
- [ ] Make sure can change apply button link text - if url is entered, text should be required
- [ ] Make sure the colleges are listed before the related links in the sidebar

Other issue:
- blt ds --site=admissions.uiowa.edu
- go to https://admissions.uiowa.ddev.site/academics/health-sci and make sure pager is removed
